package.json

```diff
@@ -20,6 +20,8 @@
     "run:hoprd:xdai": "lerna exec --scope @hoprnet/hoprd -- yarn start --host=0.0.0.0:9080 --adminPort=2990 --restPort=2991 --password='xDai' --announce --data=/tmp/xdai --identity=/tmp/xdai-identity --testNoAuthentication",
     "run:hoprd:xdai:protected": "lerna exec --scope @hoprnet/hoprd -- yarn start --host=0.0.0.0:9080 --adminPort=2990 --restPort=2991 --password='xDai' --announce --data=/tmp/xdai --identity=/tmp/xdai-identity --apiToken=xD4!-N3+w0rK",
     "run:hoprd:mumbai": "lerna exec --scope @hoprnet/hoprd -- yarn start --host=0.0.0.0:9070 --adminPort=2980 --restPort=2981 --provider=https://polygon-mumbai.infura.io/v3/${HOPRD_INFURA_KEY} --password='mumbai' --announce --data=/tmp/mumbai --identity=/tmp/mumbai-identity --testNoAuthentication",
+    "run:hoprd:goerli": "lerna exec --scope @hoprnet/hoprd -- yarn start --host=0.0.0.0:9060 --adminPort=2970 --restPort=2971 --provider=https://goerli.infura.io/v3/${HOPRD_INFURA_KEY} --password='goerli' --announce --data=/tmp/goerli --identity=/tmp/goerli-identity --testNoAuthentication",
+    "run:hoprd:goerli:protected": "lerna exec --scope @hoprnet/hoprd -- yarn start --host=0.0.0.0:9060 --adminPort=2970 --restPort=2971 --provider=https://goerli.infura.io/v3/${HOPRD_INFURA_KEY} --password='goerli' --announce --data=/tmp/goerli --identity=/tmp/goerli-identity --apiToken=g03rL!-N3+w0rK",
```

Comment:
Why do we need this?

Reply:
Short-hand commands. Think of them as snippets for anyone to use.